.appveyor/build_libedit.bat

```diff
@@ -75,22 +75,37 @@ if not exist "%BUILDDIR%\wineditline-%LIBEDIT_VERSION%" (
 2>nul rd /S /Q "%BUILDDIR%\wineditline-%LIBEDIT_VERSION%\include"
 mkdir "%BUILDDIR%\wineditline-%LIBEDIT_VERSION%\build"
 
-"%MSYSDIR%\bin\bash" --login -i -c "cd \"$BUILDDIR_M/wineditline-$LIBEDIT_VERSION/build\" && $CMAKE_BIN -DLIB_SUFFIX=32 -G \"MSYS Makefiles\" .."
+if exist "%VS90COMNTOOLS%..\..\VC\bin\vcvars32.bat" (
+    call "%VS90COMNTOOLS%..\..\VC\bin\vcvars32.bat"
+)
+
+cd "%BUILDDIR%\wineditline-%LIBEDIT_VERSION%\build" && %CMAKE_BIN% -DLIB_SUFFIX=32 -DMSVC_USE_STATIC_RUNTIME=ON -DCMAKE_BUILD_TYPE=Release -G "NMake Makefiles" ..
 if not %ERRORLEVEL% == 0 (
     echo "Failed to generate makefiles for libedit"
     exit /b 1
 )
-"%MSYSDIR%\bin\bash" --login -i -c "cd \"$BUILDDIR_M/wineditline-$LIBEDIT_VERSION/build\" && make"
+nmake
 if not %ERRORLEVEL% == 0 (
     echo "Failed to compile libedit"
     exit /b 1
 )
-"%MSYSDIR%\bin\bash" --login -i -c "cd \"$BUILDDIR_M/wineditline-$LIBEDIT_VERSION/build\" && make install"
+nmake install
 if not %ERRORLEVEL% == 0 (
     echo "Failed to install libedit"
     exit /b 1
 )
-"%MSYSDIR%\bin\bash" --login -i -c "mkdir -p \"$BINARIESDIR_M/libedit\" && cd \"$BUILDDIR_M/wineditline-$LIBEDIT_VERSION/\" && cp -rfv bin32 include lib32 \"$BINARIESDIR_M/libedit/\""
+2>nul mkdir "%BINARIESDIR%\libedit" && cd "%BUILDDIR%\wineditline-%LIBEDIT_VERSION%"
+if %ERRORLEVEL% == 0 (
+  for %%f in (bin32 include lib32) do (
+    robocopy /S %%f "%BINARIESDIR%\libedit\%%f"
```

Comment:
Just curious: Any reason to use robocopy instead xcopy ?

Reply:
No particular reason other than I read in multiple places that since Windows Vista `xcopy` seems to be regarded as deprecated as it raises memory errors with large files, and it should be replaced by `roocopy`, which is now a standard part of Windows OS; see for example https://ss64.com/nt/xcopy.html

Reply:
@ptosco Thanks for pointer... Looks like we need to change in other place also as PBS uses xcopy in file transfer, but that's another story...